desktop/windows/Dockerfile

```diff
@@ -24,6 +24,12 @@ RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache \
     --installPath C:\BuildTools \
      || IF "%ERRORLEVEL%"=="3010" EXIT 0
 
+# Install Google Root R1 cert so pub.dartlang.org stays working
+
+ADD https://pki.goog/repo/certs/gtsr1.pem C:/TEMP/gtsr1.pem
+RUN powershell.exe -Command \
+        Import-Certificate -FilePath C:\TEMP\gtsr1.pem -CertStoreLocation Cert:\LocalMachine\Root
```

Comment:
Do you think it should be on the top for better layer caching? Or you'd prefer to update it frequently?

Reply:
Ah I'm still a bit new to Docker, I'm don't quite understand what the position in the Docker file / layers has to do with caching but yeah it can be moved anywhere you prefer for those considerations. And I picked the Root cert specifically so it should auth all their front line certs they seem to generate every few months, this one is good till 2027 I think.

Just let me know where you'd prefer this and I'll move it. You want it at the end of the file? or top?

Reply:
I'll move it myself.

Docker images has layers and each one of the layers depend on the previous one. These layers are stored independently in the registry. The idea here is to try to try to put instructions (`RUN`s, `ADD`s, etc.) ideally in the order of less probability of changes. For example here, the certificate won't likely change so we can put it before `RUN`. In this case even if the `RUN` will change the image, layer with the certificate will be reused.

Reply:
ah cool, thanks!